README.md

```diff
@@ -1,34 +1,93 @@
-# Membrane Multimedia Framework: Template Plugin
+# Membrane FFmpeg SWScale plugin
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_ffmpeg_swscale_plugin.svg)](https://hex.pm/packages/membrane_ffmpeg_swscale_plugin)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_ffmpeg_swscale_plugin/)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_ffmpeg_swscale_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_ffmpeg_swscale_plugin)
 
-This repository contains a template for new plugins.
+Plugin performing video scaling, using SWScale module of [FFmpeg](https://www.ffmpeg.org/) library.
 
-Check out different branches for other flavours of template.
-
-It is part of [Membrane Multimedia Framework](https://membraneframework.org).
+It is a part of [Membrane Multimedia Framework](https://membraneframework.org).
 
 ## Installation
 
-The package can be installed by adding `membrane_template_plugin` to your list of dependencies in `mix.exs`:
+Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
 ```elixir
-def deps do
-  [
-    {:membrane_template_plugin, "~> 0.1.0"}
-  ]
+{:membrane_ffmpeg_swscale_plugin, "~> 0.1.0"}
+```
+
+You also need to have [FFmpeg](https://www.ffmpeg.org/) library installed.
+
+## Sample usage
+Scaler needs input in the YUV420p format. It processes one frame at a time. The element also requires
+getting caps with video width and height. To meet all requirements either `Membrane.Element.RawVideo.Parser` or `Membrane.H264.FFmpeg.Decoder`
+can be used before Scaler in the pipeline.
```

Comment:
This seems a bit too specific - any decoder will do the job. I would rephrase it that scaler needs raw, parsed video frames so it either requires `RawVideo.Parser` or some decoder preceding it within pipeline and here you can name `H264` as an example